### PR TITLE
Python analysis files

### LINKED ISSUE #2: Update readme for this repo
This issue can be closed when the readme reflects the content intended for this paper and links to the appropriate location for the input files, scripts, and other items that generate the data in the paper.

### LINKED ISSUE #3: Add a section of this repo for the results (output files and images)
This is probably the repo where output files should go (from moltres runs). This issue can be closed when there is a place in this repo for those results (and that place is noted in the readme for this repo).

### PR DESCRIPTION
**Pull request by smpark7**
Closes #2, closes #3 

This PR adds:
- All relevant csv output data from Moltres required to generate results and plots in the manuscript
- csv data from the CNRS benchmark paper [1] required to calculate the discrepancy values for Moltres' results
- python scripts to process the csv data and generate relevant results and plots in the manuscript
- Update base directory readme
- Add readme for python directory about the Moltres output data, how to run the python scripts, and the python output txt and png files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,22 +24,9 @@ comments and the corresponding edits. Each new round
...
-``\documentclass[review]{elsarticle}``
-
-To view the article in a "final print" format, modify the first line of
-``main.tex`` file so that it reads
-
-``\documentclass[3p, twocolumn]{elsarticle}``
-
-``elsarticle`` supports other formats that can be read about
-[here](https://www.elsevier.com/__data/assets/pdf_file/0008/56843/elsdoc-1.pdf)
+The `python` directory holds all output data files from the Moltres simulations
```
Comment (nsryan2) [inline README.md]:
```suggestion
The `python` directory holds all of the output data files from the Moltres simulations
```

File: python/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- Pandas
+- Matplotlib
+- Scipy
+
+The python scripts generate txt files which contain the average discrepancy
+values for Moltres for the given observables. e.g. ```vel-field.py```
+generates ```vel-field.txt``` which contains the average discrepancy values
+for ux and uy along centerlines AA' and BB'.
+
+The python scripts also generate the plots that we use in the manuscript.
```
Comment (nsryan2) [inline python/README.md]:
```suggestion
```

File: python/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+python3 vel-field.py
+```
+
+Required python packages include:
+- Numpy
+- Pandas
+- Matplotlib
+- Scipy
+
+The python scripts generate txt files which contain the average discrepancy
```
Comment (nsryan2) [inline python/README.md]:
```suggestion
The python scripts generate the plots used in the manuscript and txt files, which contain the average discrepancy
```

**Review by nsryan2 — COMMENTED** [review]
This is likely a problem with my device (and I'm examining it more), but I wasn't able to generate the pdf with the instructions from the README. I will let you know though, hopefully by coffee chat tomorrow if I really can't get it. Other than that, I was able to get all of the scripts to work and it looked good.

**Comment by smpark7** [comment]
Thanks @nsryan2! What error messages are you getting when you run `make`?

**Comment by nsryan2** [comment]
I had an old version of pifont, which wasn't playing nicely, but I got it up and working now!

**Comment by smpark7** [comment]
Thanks again for your review!

**Pull request merged**